src/quickstart.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+set -euo pipefail
+
+OWNER=microshift-io
+REPO=microshift
+BOOTC_ARCHIVE=""
+
+LVM_DISK="/var/lib/microshift-okd/lvmdisk.image"
+VG_NAME="myvg1"
```

Comment:
Should this be overridable? Maybe we can use another value, like `ushiftvg1` or something